tests/unit/setup-unit-test.ts

```diff
@@ -3,7 +3,7 @@ import process from "node:process";
 import { exec } from "node:child_process";
 import { promisify } from "node:util";
 import { rm, mkdir, copyFile } from "node:fs/promises";
-import { getToolVersion } from "./utils";
+import { getToolVersion } from "../../vite.config.utils";
```

Comment:
Is this the base name for it? Why not `tests/getToolVersion.ts` or something?

Reply:
Renamed in e692cf4675eb7ccf870c3d73c1df16f09988653c .